src/pages/Login/Login.scss

```
@@ -0,0 +1,119 @@
+@import url('https://fonts.googleapis.com/css2?family=Nanum+Gothic:wght@400;700;800&display=swap');
+
+.loginWrap {
+  text-decoration: none;
+  color: inherit;
+  font-family: 'Nanum Gothic', sans-serif;
+  .loginHeader {
+    height: 55px;
+    .loginHeaderBox {
+      position: fixed;
+      width: 100%;
```

css 선택자 간에 한 줄씩 띄워주세요! 
```suggestion
.loginWrap {
  text-decoration: none;
  color: inherit;
  font-family: 'Nanum Gothic', sans-serif;
  
  .loginHeader {
    height: 55px;
    
    .loginHeaderBox {
      position: fixed;
      width: 100%;
```

---

완료